old-faithful-proto/proto/old-faithful.proto

```diff
@@ -90,3 +92,34 @@ message GetResponseError {
   GetResponseErrorCode code = 1;
   string message = 2;
 }
+
+
+message StreamBlocksRequest {
+  uint64 start_slot = 1;
+  optional uint64 end_slot = 2;  // If not specified, continues indefinitely
+  optional StreamBlocksFilter filter = 3;
+}
+
+message StreamBlocksFilter {
+  repeated string account_include = 1;  // Filter blocks/txns mentioning these accounts
+  optional bool include_transactions = 2;  // Include full transaction details
+  optional bool include_accounts = 3;  // Include account updates
```

Comment:
what does this do?

Reply:
I was just following the dragon's mouth api: https://docs.triton.one/project-yellowstone/dragons-mouth-grpc-subscriptions#subscribe-to-blocks

Reply:
do you know if that means "filter transactions that include these accounts" or "actually include the actual accounts state"